src/app/server/CommissioningWindowManager.cpp

```diff
@@ -119,6 +119,9 @@ void CommissioningWindowManager::Shutdown()
 void CommissioningWindowManager::ResetState()
 {
     mUseECM = false;
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
```

Comment:
Changes approved on my side but I would really like to also have @bzbarsky-apple to take a look.